src/main/java/org/matsim/project/prebookingStudy/run/dummyTraffic/DvrpBenchmarkTravelTimeModuleFixedTT.java

```diff
@@ -0,0 +1,45 @@
+package org.matsim.project.prebookingStudy.run.dummyTraffic;
+
+import com.google.inject.Inject;
+import com.google.inject.Key;
+import com.google.inject.name.Names;
+import org.matsim.api.core.v01.TransportMode;
+import org.matsim.contrib.dvrp.run.DvrpConfigGroup;
+import org.matsim.contrib.dvrp.trafficmonitoring.DvrpOfflineTravelTimes;
+import org.matsim.contrib.dvrp.trafficmonitoring.DvrpTravelTimeModule;
+import org.matsim.contrib.dvrp.util.TimeDiscretizer;
+import org.matsim.core.controler.AbstractModule;
+import org.matsim.core.router.util.TravelTime;
+
+import java.net.URL;
+
+public class DvrpBenchmarkTravelTimeModuleFixedTT extends AbstractModule {
+    @Inject
+    private DvrpConfigGroup dvrpCfg;
+
+    private final double travelTimeOverEstimation;
+
+    public DvrpBenchmarkTravelTimeModuleFixedTT(double travelTimeOverEstimation) {
+        this.travelTimeOverEstimation = travelTimeOverEstimation;
+    }
+
+    public void install() {
+        if (dvrpCfg.getInitialTravelTimesFile() != null) {
+            addTravelTimeBinding(DvrpTravelTimeModule.DVRP_ESTIMATED).toProvider(() -> {
+                URL url = dvrpCfg.getInitialTravelTimesUrl(getConfig().getContext());
+                var timeDiscretizer = new TimeDiscretizer(getConfig().travelTimeCalculator());
+                var linkTravelTimes = DvrpOfflineTravelTimes.loadLinkTravelTimes(timeDiscretizer, url);
+                return DvrpOfflineTravelTimes.asTravelTime(timeDiscretizer, linkTravelTimes);
+            }).asEagerSingleton();
```

Comment:
Perhaps this can be removed. AFAIK we do not use this option?

Reply:
Yes, that is correct. We don't have any TravelTime file for this study. We can remove this.